### ISSUE TITLE
[Request] Folder & Tabs in drawer simultaneously

### ISSUE THREAD

**Issue by doobydooby72** [otakuhqz/Omega#61]
Hi just discovered your launcher and it seems really awesome... Almost exactly what im looking for. The one thing i really love about nova launcher is its ability to have both tabs and folders in the drawer. I noticed your app has these options but is only able to utilise one at a time. Would be awesome if you could have both optionally active at the same time. 
Thank you

**Comment by Donnnno** [comment]
It would be great to have tabs and folders indeed! That's the only thing that keeps me going back to Nova :S

**Comment by tdadvocate** [comment]
I have to agree. This feature is desparately needed. I love your launcher but this missing feature makes me keep going back to Nova no matter what. I have over 300 apps on my device. I don't want to scroll through a million tabs or folders alone. Having both is very necessary for organising all of these different apps with similar major categories but slightly different subcategories.

**Comment by FreezyExp** [comment]
Same, tabs are a nice top level separator, i usually have just two.
Three or more is understandable, with looping around for ease of acces (for example: swipe left for work, right for theming)

Folders are a nice way to organize a group of related apps. Like social, camera, audio.

**Comment by saulhdev** [comment]
added in commit 723526e24b2c17202088669abd2a1cce5f09a75d
